bundle-workflow/Jenkinsfile

```diff
@@ -7,6 +7,11 @@ pipeline {
         jdk "JDK14"
         maven "maven-3.8.2"
     }
+    triggers {
+      parameterizedCron '''
+        H */2 * * * %INPUT_MANIFEST=1.1.0/opensearch-1.1.0.yml
```

Comment:
What is `H` in this?

Reply:
I do not understand cron much like regex, but there is this great tool [cron.help](https://cron.help/#*/2_*_*_*_*) that you could plug values into see what it might do.

Also - this is our existing schedule that runs every 2 hours, but is not in source control.

Reply:
For every 2 hours shouldn't it be `0 */2 * * *`?

Reply:
I think the question is "what is the value of 'H' here"?

Reply:
I figured it out - https://www.jenkins.io/doc/book/pipeline/syntax/#cron-syntax

Reply:
That explains it. Thanks @bbarani